plugin/resolvergen/testdata/singlefile_enable_rewrite/out/model.go

```diff
@@ -0,0 +1,13 @@
+package customresolver
+
+import "context"
+
+type Resolver struct{}
+
+type QueryResolver interface {
```

Comment:
Why are these interfaces being generated here? Aren't these normally inside the server executable?

Reply:
@clayne11 You are right! The test are supposed to include only one `model.go` with necessary interfaces (that are usually in the server.go). They supply it this way only for the purpose of a compact unit test I think.